ui/.graphclientrc.yml

```
@@ -0,0 +1,9 @@
+# .graphclientrc.yml
+sources:
+  - name: FleekNFA
+    handler:
+      graphql:
+        endpoint: https://api.thegraph.com/subgraphs/name/emperororokusaki/flk-test-subgraph #replace for nfa subgraph
+
+documents:
+  - ./queries.graphql
```

Maybe is better have a folder for graphql files and get all files
```suggestion
  - ./graphql/*/*.graphql
```